### DISCUSSION TITLE
Streamer/DJ Account - Stream URL

### DISCUSSION THREAD

**Discussion by woshjar** [AzuraCast/AzuraCast#3737] [Q&A]
If I setup a Streamer/DJ Account, how do I find the output of that to play it directly, not listening through the main station URL?

Thanks in advance!!

**Comment by BusterNeece** [comment]
AzuraCast uses our Liquidsoap AutoDJ to handle incoming DJ connections so that it can support multiple accounts, more flexible authentication, and also transcode the broadcasted audio into whatever formats are used by the station.

As a result, the composited radio signal that's broadcast out to streams includes live broadcasts when they're live and the AutoDJ otherwise.

For advanced uses, we do define a variable in our Liquidsoap script called `radio_without_live` that lets you separately handle the live and non-live radio signals, but this is a power-user function that we don't officially support.

**Comment by woshjar** [comment]
I’ve sent this to the guy that runs my server but not sure if I explained it right. So the main listener output in this case is; https://dj.eldargaming.co.uk/radio/8000/radio.mp3?1611908567 & to stream to it someone is connecting via Icecast & BUTT using the domain name or IP & a password, how can I directly listen to this feed so that if the external streamer isn’t running I can’t open the link it just goes dead? Thanks
